Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM kbase/kb_jre
+
+# These ARGs values are passed in via the docker build command
+ARG BUILD_DATE
+ARG VCS_REF
+ARG BRANCH=develop
+
+COPY deployment/ /kb/deployment/
```

Comment:
Can we make this a multistage build before we merge.

Reply:
Its been converted to multistage dockerhub builds